packages/bpk-component-nudger/src/BpkNudger.module.scss

```diff
@@ -37,6 +36,16 @@
     @include typography.bpk-text;
     @include typography.bpk-heading-5;
 
+    &::-webkit-outer-spin-button,
+    &::-webkit-inner-spin-button {
+      margin: 0;
+      appearance: none;
+    }
+
+    &[type='number'] {
+      appearance: textfield;
```

Comment:
Hides the little number widgets 
![Screenshot 2024-08-06 at 15 00 14](https://github.com/user-attachments/assets/f71879b2-fd61-446d-8f35-1f04c704919e)